scripts/ai_dev_runner.sh

```
@@ -0,0 +1,323 @@
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+# Configuration
+FLUTTER_VERSION="3.24.5"
+COVERAGE_THRESHOLD=80
```

Shellcheck (0.10.0)

[warning] COVERAGE_THRESHOLD appears unused. Verify use (or export if used externally).

(SC2034)